scripts/genproto.sh

```diff
@@ -60,6 +60,7 @@ for dir in ${DIRS}; do
 		protoc --gofast_out=plugins=grpc,import_prefix=github.com/coreos/:. -I=.:"${GOGOPROTO_PATH}":"${COREOS_ROOT}":"${GRPC_GATEWAY_ROOT}/third_party/googleapis" *.proto
 		sed -i.bak -E "s/github\.com\/coreos\/(gogoproto|github\.com|golang\.org|google\.golang\.org)/\1/g" *.pb.go
 		sed -i.bak -E 's/github\.com\/coreos\/(errors|fmt|io)/\1/g' *.pb.go
+		sed -i.bak -E 's/import math \"github\.com\/coreos\/math\"//g' *.pb.go
```

Comment:
This is needed when proto file has `double` field, the generated Go code inserts `import math "github.com/coreos/math"` line.